pyanaconda/startup_utils.py

```diff
@@ -175,6 +176,22 @@ def set_storage_checker_minimal_ram_size(display_mode):
     )
 
 
+def fallback_tui_if_gtk_ui_not_available(anaconda):
```

Comment:
Maybe `fallback_to_tui_if_gtk_ui_is_not_available` ? Its already long, so that won't hurt & it looks better. :)

Reply:
Fixed.